.gitignore

```diff
@@ -32,3 +32,8 @@ dockstore-webservice/nbactions.xml
 # confidential testing files
 secrets.tar
 dockstore-integration-testing/src/test/resources/confidential_test.txt
+dockstore-integration-testing/src/test/resources/config_file.txt
+dockstore-integration-testing/src/test/resources/db_dump.sql
```

Comment:
db_dump was the original db_confidential_dump, I just forgot to remove it from the gitignore.  I'll remove it along with the confidential_test.txt and add some comments.